modules/Book.js

```diff
@@ -0,0 +1,20 @@
+// eslint-disable-next-line import/prefer-default-export
+export class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+    this.store = window.localStorage;
+  }
+
+  static books = [];
+
+  addNewBook() {
+    Book.books.push({ title: this.title, author: this.author });
+    this.store.setItem('books', JSON.stringify(Book.books));
+  }
+
+  static removeBook(title) {
+    Book.books = Book.books.filter((book) => book.title !== title);
+    this.store.setItem('books', JSON.stringify(Book.books));
+  }
+}
```

Comment:
- [ ] The book removing method is not working. Please, try to fix it:
![2022-02-14 14 25 36](https://user-images.githubusercontent.com/49064106/153846483-44431a53-4279-4589-bbca-429f3c9ace3b.gif)

Reply:
Done

Reply:
Done